src/main/java/edu/harvard/iq/dataverse/api/exposedsettings/Setting.java

```diff
@@ -0,0 +1,38 @@
+package edu.harvard.iq.dataverse.api.exposedsettings;
+
+public class Setting<T> extends SettingItem {
+    /**
+     * Setting names for group {@link edu.harvard.iq.dataverse.api.exposedsettings.SettingGroup#GROUP_NAME_DATAVERSE}
+     */
+    public static final String SETTING_NAME_FQDN = "fqdn";
```

Comment:
Does the SPA need `fqdn`? If not, should we expose `siteUrl` instead? We plan to deprecate `fqdn` in favor of `siteUrl`. See this issue:

- #6636

Reply:
I have added _fqdn_ because it is a setting that was already exposed with the old mechanism (API info dedicated endpoint) so we make the new mechanism support everything from the old one. Currently the SPA does not need to read either _fqdn_ or _siteUrl_.

I wouldn't consider the SPA to be the only consumer of this endpoint. Its goal is to be a general endpoint for any API consumer interested in reading public settings.

Seeing the intention to deprecate fqdn, maybe a good approach is to remove it from the old and new mechanism at the same time. What do you think?